clang/lib/Format/Format.cpp

```diff
@@ -1673,10 +1678,15 @@ FormatStyle getLLVMStyle(FormatStyle::LanguageKind Language) {
   LLVMStyle.InsertBraces = false;
   LLVMStyle.InsertNewlineAtEOF = false;
   LLVMStyle.InsertTrailingCommas = FormatStyle::TCS_None;
-  LLVMStyle.IntegerLiteralSeparator = {
-      /*Binary=*/0,  /*BinaryMinDigits=*/0,
-      /*Decimal=*/0, /*DecimalMinDigits=*/0,
-      /*Hex=*/0,     /*HexMinDigits=*/0};
+  LLVMStyle.IntegerLiteralSeparator = {/*Binary=*/0,
+                                       /*BinaryMinDigits=*/0,
+                                       /*BinaryMaxDigitsNoSeparator=*/-1,
+                                       /*Decimal=*/0,
+                                       /*DecimalMinDigits=*/0,
+                                       /*DecimalMaxDigitsNoSeparator=*/-1,
+                                       /*Hex=*/0,
+                                       /*HexMinDigits=*/0,
+                                       /*HexMaxDigitsNoSeparator=*/-1};
```

Comment:
```suggestion
  LLVMStyle.IntegerLiteralSeparator = {};
```
Better to use `0` as the default to mean "leave" or "disabled".